src/main/scripts/src/app/app.component.ts

```diff
@@ -33,8 +33,8 @@ export class AppComponent implements OnInit {
 
   public notificationsOptions = {
     position: ['top'],
-    timeOut: 5000,
-    showProgressBar: false,
+    timeOut: 1000,
+    showProgressBar: true,
```

Comment:
What is the progress bar good for?

Reply:
The main reason was that it looks good.
On the practical side however, by having an animation it brings more attention to the popup. Lets users know how long until it disappears, and lets them know that by hovering above the popup the timer stops (..so they can read the message).